### PR TITLE
Question.html

### PR DESCRIPTION
**Pull request by aishwarya-bm**
added questions page
-shows category title, current score & question number, Q & options, buttons to navigate to next Q & quit game.
-dummy nav to home page from index.html

NOTE: changed domain name preview link doesn't work

https://user-images.githubusercontent.com/66695068/153708975-d3d4ba61-c73a-49c5-9667-0109ccf4de91.mp4

### CODE REVIEW

File: stylesheets/question.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.question-container {
+    display: flex;
+    flex-direction: column;
```
Comment (kjais1720) [inline stylesheets/question.css]: You can use the utility class (imported from the CL) for this

File: stylesheets/question.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    margin: 3rem auto;
+}
+
+.question-score-info {
+    display: flex;
+    justify-content: space-between;
+}
+
+.question-current {
+    font-size: 1.25rem;
```
Comment (kjais1720) [inline stylesheets/question.css:7]: Try to use variables for font sizes wherever you can

File: stylesheets/question.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    text-align: left;
+    margin-top: 2rem;
+}
+
+.question-options-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```
Comment (kjais1720) [inline stylesheets/question.css]: Use utility class for the same

**Review by kjais1720 — CHANGES_REQUESTED** [review]

File: stylesheets/question.css [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    padding: 1rem;
+    box-shadow: none;
+    cursor: pointer;
+    border: 1px solid var(--contrast-light);
+}
+
+ .question-actions {
+    display: flex;
+    justify-content: space-between;
+}
```
Comment (Tanjum786) [inline stylesheets/question.css:28]: code looks neat. Just small suggestion you can make common class for display flex you are using it in multiple time so.

File: stylesheets/question.css (hunk 1 above)
Comment (aishwarya-bm) [reply stylesheets/question.css]: oh yes, made the changes

File: stylesheets/question.css (hunk 2 above)
Comment (aishwarya-bm) [reply stylesheets/question.css:7]: i dont have util for this font size, so I might not make changes here

File: stylesheets/question.css (hunk 3 above)
Comment (aishwarya-bm) [reply stylesheets/question.css]: made changes

File: stylesheets/question.css (hunk 4 above)
Comment (aishwarya-bm) [reply stylesheets/question.css:28]: made changes

**Review by Tanjum786 — APPROVED** [review]

**Review by kjais1720 — APPROVED** [review]

**Pull request merged**